### PR TITLE
Mde/simulation fix

### PR DESCRIPTION
**Pull request by dmarienko**

### CODE REVIEW

**Comment by dmarienko** [comment]
I'd propose to merge this into main

**Comment by yuriy-arabskyy** [comment]
thanks, I will read through it in a couple minutes

File: src/qubx/data/helpers.py [hunk 1]
```diff
@@ -189,23 +190,27 @@ def _handle_symbols_data_from_to(
...
+                # self._data[k] = srows(c, self._data[k], keep="first")
+                self._data[k] = srows(c, self._data[k], keep="last")
 
-        # - part intervals
+        # - post intervals
         if self._stop and _stop > self._stop:
             _smbs = list(self._data.keys())
-            logger.debug(f"Updating {len(_smbs)} symbols after interval {self._stop} : {_stop}")
-            _after = self._load_candle_data(_smbs, self._stop - _dtf, _stop, self._data_timeframe)
+            logger.debug(f"(InMemoryCachedReader) Updating {len(_smbs)} symbols post interval {self._stop} : {_stop}")
```
Comment (yuriy-arabskyy) [inline src/qubx/data/helpers.py:211]: I suggest adding class name into the logger configuration. But let's do this later in a separate refactoring.

**Review by yuriy-arabskyy — APPROVED** [review]

**Comment by yuriy-arabskyy** [comment]
looks good, let's merge it

**Pull request merged**